tests/playbooks/roles/install-csi-cinder/defaults/main.yaml

```diff
@@ -0,0 +1,10 @@
+---
+github_pr: 123
```

Comment:
isn't this env  specific , not related to specific plugin, so we could reuse

Reply:
yes, but this comes from tests/ci-csi-cinder-e2e.sh, each playbook has different template
so we should set here as well?

Reply:
Im not sure actually, not good in ansible . many parts of setting up env like (create-vm, env set) did in ci-csi-cinder-e2e.sh and here are same as in occm test , so just wondering if anything can be done to be reused in all tests . But not blocker for now, something can be done as improvement if possible.

May be @lingxiankong can suggest if possible or not.